src/ui/server/controllers/insurance/check-your-answers/your-buyer/save-and-back/index.test.ts

```diff
@@ -0,0 +1,42 @@
+import { Request, Response } from '../../../../../../types';
+import { post } from '.';
+import { ROUTES } from '../../../../../constants';
+import { mockReq, mockRes, mockApplication } from '../../../../../test-mocks';
+
+const { INSURANCE_ROOT, ALL_SECTIONS } = ROUTES.INSURANCE;
+
+describe('controllers/insurance/check-your-answers/your-buyer/save-and-back', () => {
+  let req: Request;
+  let res: Response;
+
+  beforeEach(() => {
+    req = mockReq();
+    res = mockRes();
+
+    res.locals.application = mockApplication;
+  });
+
+  afterAll(() => {
+    jest.resetAllMocks();
+  });
+
+  describe('post - save and back', () => {
```

Comment:
Just noticed these in other controllers, we don't need this describe wrapper. Shall we do in simplify all of these in a future PR?

Reply:
future PR please

Reply:
https://ukef-dtfs.atlassian.net/browse/EMS-1317